src/tests/ftest/io/macsio_test.yaml

```diff
@@ -34,13 +34,19 @@ macsio:
   part_size: 10M
   avg_num_parts: 2.5
   num_dumps: 2
+  debug_level: 1
+macsio_timeout:
+  test_macsio: 20
+  test_macsio_daos_vol: 90
 mpi: !mux
   mpich:
-    mpi_type: "mpich"
+    job_manager_class_name: Mpirun
+    job_manager_mpi_type: mpich
     macsio_path: /usr/lib64/mpich/bin
     plugin_path: /usr/lib64/mpich/lib
   openmpi:
-    mpi_type: "openmpi"
+    job_manager_class_name: Orterun
+    job_manager_mpi_type: openmpi
```

Comment:
In case of class Orterun, mpi type is always openmpi. So can we just specify the class name and save ourselves from mentioning an extra line of mpi type in all the test yamls?

Reply:
It is not required in this case.  It will be passed to the `general_utils.py get_job_manager_class()` method but not used:
```
    if name == "Mpirun":
        manager = manager_class(job, subprocess=subprocess, mpitype=mpi)
    else:
        manager = manager_class(job, subprocess=subprocess)
```

Reply:
Yeah, so can we remove the line?